lms/djangoapps/instructor/static/instructor/ProblemBrowser/components/ReportStatus/ReportStatus.jsx

```diff
@@ -0,0 +1,55 @@
+/* global gettext */
+import { Icon } from '@edx/paragon';
+import classNames from 'classnames';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+const ReportStatus = ({ error, succeeded, inProgress, reportPath }) => {
+  const progressMessage = (
+    <div className="msg progress">
+      {gettext('Your report is being generated...')}
+      <Icon hidden className={['fa', 'fa-refresh', 'fa-spin', 'fa-fw']} />
+    </div>
+  );
+
+  const successMessage = (
+    <div className="msg success">
+      {gettext('Your report has being successfully generated.')}
+      <a href={reportPath}>
+        <Icon hidden className={['fa', 'fa-link']} />
+        {gettext('View Report')}
+      </a>
+    </div>
+  );
+
+  const errorMessage = (
+    <div className={classNames('msg', { error })}>
+      {error && `${gettext('Error')}: `}
+      {error}
+    </div>
+  );
+
+  return (
+    <div className="report-generation-status">
```

Comment:
I think we may want `aria-live="polite"` here too, so that changes get announced to users with screen readers, at appropriate times?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

Reply:
Done.